src/components/NewWorkspaceModal.js

```diff
@@ -170,8 +171,8 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
   const isBillingProjectApplicable = project => {
     // Only support cloning a workspace to the same cloud environment.
     return Utils.cond(
-      [!!cloneWorkspace && !!cloneWorkspace.azureContext, () => isAzureBillingProject(project)],
-      [!!cloneWorkspace && !cloneWorkspace.azureContext, () => !isAzureBillingProject(project)],
+      [!!cloneWorkspace && isAzureWorkspace(cloneWorkspace), () => isAzureBillingProject(project)],
+      [!!cloneWorkspace && isGoogleWorkspace(cloneWorkspace), () => !isAzureBillingProject(project)],
```

Comment:
perhaps test that it's a google billing project directly?

Reply:
Sure, done.